CougSat1-IHU/src/subsystems/Payload.h

```diff
@@ -3,19 +3,151 @@
 
 #include "Subsystem.h"
 
-#include <mbed.h>
+#include "mbed.h"
+// Includes the BME280 driver inside the CISLibrary for
+// use by getPressure(), getTemperature(), and getHumidity().
+#include "../../../CISLibrary/drivers/BME280/BME280.h"
```

Comment:
#include <mbed.h>
#include <BME280/BME280.h>

Don't need the comment to explain what #include does and PlatformIO has the CISLibrary paths setup correctly. If it doesn't work, follow the readme.